File: LC-1323/LC-1323.js
/**
 * @param {number} num
 * @return {number}
 */
var maximum69Number  = function(num) {
    /**
     * Concepts: Number to String
     */
    
    let numStr = num.toString();
    for (let i = 0; i < numStr.length; ++i) {
        if (numStr[i] === '6') {
            return +(numStr.substring(0, i) + '9' + numStr.substring(i + 1));
        }
    }
    return num
};

File: LC-1323/LC-1323.cpp
#include <string>

using namespace std;

class Solution {
public:
    /**
     * Concepts: Convert integer to string 
     */
    int maximum69Number (int num) {
        string numStr = to_string(num);
        for (int i = 0; i < numStr.length(); ++i) {
            if (numStr[i] == '6') {
                numStr[i] = '9';
                break;
            }
        }
        return stoi(numStr);
    }
};